### DISCUSSION TITLE
How to override properties of sub-components of a component?

### DISCUSSION THREAD

**Discussion by chrissyast** [vuetifyjs/vuetify#17058] [Q&A]
I am using `v-select`. Out of the box, `v-select`s contain a `v-text-field` which has `readonly` set to true but I want to override this.

The key part of the source code seems to be at 

```js
// \node_modules\vuetify\lib\components\VSelect\VSelect.mjs
const vTextFieldRef = ref();
// ......
return _createVNode(VTextField, _mergeProps({
        "ref": vTextFieldRef
      }, textFieldProps, {
        // ...
        "readonly": true,
        // ...
      }),
```

I have created a component which extends VSelect but I don't know how to set the VTextField to `readonly: false`. 
`vTextFieldRef` is initialised with no initial value and never seems to be set, so I don't know if it's possible to find it with this.$refs[something] (no refs show up in the Vue devtools either)

**Comment by KaelWD** [comment]
> but I want to override this

Why? Are you looking for https://vuetifyjs.com/en/components/autocompletes/ or https://vuetifyjs.com/en/components/combobox/ instead? Even if there was a way to remove readonly none of the other props are hooked up like in a combobox.

**Reply by chrissyast** [reply]
Autocomplete doesn't fit my use case because it clears the input on blurring
Combobox doesn't fit my use case because it allows the user to add a custom selection if what they type doesn't match any of the provided options
